### ISSUE TITLE
[CST] Update name of tracked item in Recent Activity section to reflect updated 5103 alert 

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#82603]
## Value Statement
**_As a_** veteran with a 5103 request
**_I want to_** see content consistent with how these requests are described in other areas (evidence pages and alerts) in Recent Activity
**_So that_** I can associate the items with previous alerts and actions I might have taken regarding 5103 requests. 

---
## Background Context

The recent activity language for 5103 requests needs to be in-line with the content produced for the evidence detail page and alert. 

Current UI 

<img width="658" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNkxCQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7f0512766c692d1e03c8789906c3d44675c4999a/ListItem1.png" alt="ListItem1.png" />

<img width="658" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNlBCQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--726f6831bb96ee7ed6a1000cfbaa04b924230f7c/ListItem2.png" alt="ListItem2.png" />

<img width="658" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNlRCQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7c1ee73379f3b6995209f05931fe4a83bb2a66ae/ListItem3.png" alt="ListItem3.png" />

## Outcome, Success Measure, KPI(S), and Tracking Link
- More 5103 waivers submitted through CST
- More 5103 notices downloaded through claim letters section

## Design
- [Figma](https://www.figma.com/design/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation-and-5103-alert?node-id=1694-15765&t=ihxrPINq7bDO71qY-4)
- New experience should conform to naming conventions used in the "header" for evidence detail pages. 

### Enablement team (if needed)
@jstrothman

## Engineering Notes:
- Put behind feature flag `cst_5103_update_enabled`
- Look to display name
- Logic would look something like this:
```
const is5103Notice =
        trackedItem.displayName === 'Automated 5103 Notice Response' || trackedItem.displayName === '5103 Notice Response' ;
```

## Out of scope
- 1P alert on Status and Files tab
- Document Request page for 5103 1P alert
- Confirmation when 5103 waiver is submitted

## Open questions
- Do we have control over the tracked item description?
- If a Veteran submits waiver via "Ask for your claim decision" today, does anything get added to the Documents Filed section?
- Is this supposed to be an eng and design ticket? confused after reading the tasks that design added

## Design Tasks
- [x] Finalize language tracked item
- [x] Update the tracked item description if possible to reflect language from the updated 5103 notice 

## Engineering Tasks

- [x] Add new logic to recent activity component
- [x] When tracked item is a closed 5103 notice the text for the recent activity list item no longer says '5103 notice response' and instead says  '5103 Evidence Notice'
- [x] When tracked item is an Automated 5103 notice the text for the recent activity list item no longer says 'Automated 5103 notice response' and instead says  '5103 Evidence Notice'
- [x] Put logic behind a feature flag `cst_5103_update_enabled`
- [x] Add unit tests

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] The 5103 description in the Recent Activity section matches the header text we use in the alert associated with the item.
- [ ] The 5103 description in the Recent Activity section no longer says '5103 notice response' and instead says  '5103 Evidence Notice'
- [ ] The 5103 description in the Recent Activity section no longer says 'Automated 5103 notice response' and instead says  '5103 Evidence Notice'
- [ ] Put behind feature flag `cst_5103_update_enabled`

**Comment by samcoforma** [comment]
@skylerschain @uxgary the Figma link is incorrect on this. Can you add the correct one as well as the language you want?

cc: @pmclaren19 @jerekshoe @jacobworrell

**Comment by pmclaren19** [comment]
> Whoops meant to update this last week. Here you go: https://www.figma.com/design/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation-and-5103-alert?node-id=1851-1961&t=44ULxttEUlNNUFBe-4
> 
> Just to confirm though, are we working this now? We don't need to update the Recent Activity section until we've updated the [Document Request page language](https://www.figma.com/design/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation-and-5103-alert?node-id=1699-17749&t=44ULxttEUlNNUFBe-4).

@skylerschain @jacobworrell pulled this ticket into backlog refinement so thats why we were looking at it. If there is another ticket that needs to be done before this one, could you please link it and mark this ticket as blocked by it? Thanks!

**Comment by skylerschain** [comment]
Yeah [here's the ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/84410) for the document request page update. We should do that first.

**Comment by pmclaren19** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/issues/84410 has been worked  and completed but with the ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/80589 we have changed the H1 title of a 5103 tracked item from `Review the list of evidence we need` to `5103 Evidence Notice`. @skylerschain should we be updating the A/C of this ticket?

**Comment by pmclaren19** [comment]
Spoke with @skylerschain and he agrees! He will update the figma design and I updated the ticket info/ ac.

**Comment by pmclaren19** [comment]
Verified in staging on user 23 and looks good! @skylerschain @jacobworrell feel free to validate and lmk if you notice anything off.
